<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>Data Dokter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NIK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNIK">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nama Lengkap</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNama">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="optLaki">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Laki-laki</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optPerempuan">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perempuan</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboSpesialis">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Anak</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kandungan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mata</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kulit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jenis Kelamin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Spesialis</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridDokter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="btnCari">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cari</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tarif</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTarif">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>136</width>
      <height>66</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 255);</string>
     </property>
     <property name="text">
      <string>Salsabila Niarno</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 255);</string>
     </property>
     <property name="text">
      <string>200511152</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 255);</string>
     </property>
     <property name="text">
      <string>R2</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="btnSimpan">
     <property name="styleSheet">
      <string notr="true">#btnSimpan{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 255);
}</string>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="styleSheet">
      <string notr="true">#btnClear{
	background-color: rgb(0, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="styleSheet">
      <string notr="true">#btnHapus{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
}
</string>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
